app/models/user.rb

```diff
@@ -43,5 +42,6 @@ class User
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable # , :omniauthable
+         :recoverable, :rememberable, :trackable, :validatable
+         # , :omniauthable
```

Comment:
Exception I think, the :omniauthable is part of the previous list, and was breaking line length. Anyway, it can probably be deleted?